src/hubbleds/components/dotplot_tutorial_slideshow/dotplot_tutorial_slideshow.vue

```diff
@@ -225,8 +224,14 @@ export default {
 
 </script>
 
-<style>
+<style scoped>
 .no-transition {
   transition: none;
 }
+
+.row {
+  width: 100%;
+  margin-left: 0;
+  margin-right: 0;
+}
```

Comment:
In looking on my browsers, this section doesn't seem necessary. Mine doesn't even use the margin values. But this does no harm. 

I think this is good to merge

Reply:
At least for me, I need these (and I actually think they should be `!important`, now that I'm looking at it again). Without them, the row div actually hangs 12px outside of the dialog (and has 12px extra space on the right).

![dotplot-scrollbars](https://github.com/cosmicds/hubbleds/assets/14281631/07308a89-54c1-4ab0-aced-4744305eac20)